monkestation/code/modules/liquids/liquid_ocean.dm

```diff
@@ -490,10 +491,12 @@ GLOBAL_LIST_INIT(the_lever, list())
 	base_lighting_alpha = 0
 	//map_generator = /datum/map_generator/ocean_generator
 	map_generator = /datum/map_generator/cave_generator/trench
+	area_flags = VALID_TERRITORY | UNIQUE_AREA | CAVES_ALLOWED | FLORA_ALLOWED | MOB_SPAWN_ALLOWED | MEGAFAUNA_SPAWN_ALLOWED
```

Comment:
Dont make these valid territory, that makes things like cults be able to get them as an area for summon